rollup/conf/config.json

```diff
@@ -76,7 +76,9 @@
       "batch_timeout_sec": 300,
       "gas_cost_increase_multiplier": 1.2,
       "max_uncompressed_batch_bytes_size": 634880
-    }
+    },
+    "chunk_proposal_interval_milliseconds": 0,
```

Comment:
why not add this under `chunk_proposer_config`? e.g. `chunk_proposer_config.propose_interval_milliseconds`

Reply:
makes sense. changed in [538d7f2](https://github.com/scroll-tech/scroll/pull/1459/commits/538d7f20098d8eb28f0693ad6fc0e7d0253d028f).